src/python/tests/core/api/test_scheduler.py

```diff
@@ -55,13 +55,13 @@ def setup_class(cls):
         asyncio.get_event_loop().run_until_complete(add_master_data())
         from subprocess import call
         call(["foglamp", "start"])
-        time.sleep(4)
+        time.sleep(14)
```

Comment:
is it machine specific or deterministic calculation?

Reply:
Value provided to remove any lag whatsoever due to async processing.